doc_test.go

```diff
@@ -85,8 +84,7 @@ type stubField struct {
 	value          []byte
 	arrayPositions []uint64
 	encodedType    byte
-	stored         bool
-	docVals        bool
+	options index.FieldIndexingOptions
```

Comment:
Indentation, `go fmt ./...`

Reply:
no need to duplicate feedback everywhere, commit is cherry-picked

Reply:
ya figured :)